src/interactions/User/AsEndUser.js

```diff
@@ -51,6 +54,28 @@ export class AsEndUser {
     return this.isLoggedIn() &&
            !!_find(this.user.notifications, {isRead: false})
   }
+
+  /**
+   * Returns a mr-text-* color for the user's displayName
+   */
+  colorCode() {
+    return mapColors(this.user.osmProfile.displayName)
+  }
+}
+
+/**
+ * Returns a mr-text-* color based off a hash of the username.
+ */
+export function mapColors(username) {
```

Comment:
Can we rename this method? `mapColors` isn't very informative, especially when used outside of this file. Maybe `usernameColor` or something.